File: Sigmath.ConvertedToPython/computing/nums.py
class Nums(object):
	def Gcd(a, b):
		while b != 0:
			temp = b
			b = a % b
			a = temp
		return a

	Gcd = staticmethod(Gcd)

	def Lcm(a, b):
		return Basic.Abs(a * b) / Nums.Gcd(a, b)

	Lcm = staticmethod(Lcm)

	def Mod(a, b):
		r = a % b
		return r + b if r < 0 else r

	Mod = staticmethod(Mod)

File: computing/nums.cs
namespace Mathian
{
    public class Nums
    {

        public static int Gcd(int a, int b)
        {
            while (b != 0)
            {
                int temp = b;
                b = a % b;
                a = temp;
            }
            return a;
        }

        public static int Lcm(int a, int b)
        {
            return Basic.Abs(a * b) / Gcd(a, b);
        }

        public static int Mod(int a, int b)
        {
            int r = a % b;
            return r < 0 ? r + b : r;
        }
    }
}
